Sources/StreamChatUI/Deprecations.swift

```diff
@@ -308,3 +308,8 @@ public extension Appearance.Images {
         set { messageDeliveryStatusRead = newValue }
     }
 }
+
+public extension CGSize {
+    @available(*, deprecated, message: "use Components.avatarThumbnailSize instead.")
+    static var avatarThumbnailSize: CGSize { CGSize(width: 40, height: 40) }
```

Comment:
In this case, this was in `ImageLoading.swift` which didn't make any sense. And because we don't have a `CGSize+Extensions.swift` file, and it also does not make sense to create one just to add a deprecation, I put it here. Ideally, we will never have public extensions in native iOS types.